File: src/MyApplication.Web.UI/src/auth/tokenStore.ts
// Access token живёт только в памяти модуля (не в localStorage/sessionStorage) —
// так он недоступен другому JS через постоянное хранилище и пропадает при
// перезагрузке страницы. Восстановление сессии после перезагрузки — через
// refresh (httpOnly cookie), см. AuthProvider.

let accessToken: string | null = null;

export function getAccessToken(): string | null {
  return accessToken;
}

export function setAccessToken(token: string | null): void {
  accessToken = token;
}

File: src/MyApplication.Android/app/src/main/kotlin/io/github/rualert/mynotesapp/data/auth/TokenStore.kt
package io.github.rualert.mynotesapp.data.auth

/**
 * Access-token живёт только в памяти процесса — на диск он не попадает, как и
 * в веб-интерфейсе (там это модульная переменная, а не localStorage). Пережить
 * перезапуск приложения ему и не нужно: сессия восстанавливается по
 * refresh-cookie, см. [AuthRepository.restoreSession].
 */
class TokenStore {

    @Volatile
    var accessToken: String? = null
}
